packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/modelservice/ModelServiceProxy.java

```diff
@@ -9,50 +11,57 @@
 import software.uncharted.terarium.hmiserver.models.modelservice.SimulateParams;
 import software.uncharted.terarium.hmiserver.models.modelservice.ModelCompositionParams;
 
+
 @RegisterRestClient(configKey = "model-service")
 @Produces(MediaType.APPLICATION_JSON)
-@Path("api/models")
 public interface ModelServiceProxy {
 	@PUT
 	Response createModel();
 
 	@POST
-	@Path("/{modelId}")
+	@Path("/api/models/{modelId}")
 	@Consumes(MediaType.APPLICATION_JSON)
 	Response addModelParts(
 		@PathParam("modelId") String modelId,
 		Graph graph
 	);
 
 	@GET
-	@Path("/{modelId}/json")
+	@Path("/api/models/{modelId}/json")
 	@Consumes(MediaType.APPLICATION_JSON)
 	Response getModelJSON(
 		@PathParam("modelId") String modelId
 	);
 
 	@POST
-	@Path("/{modelId}/simulate")
+	@Path("/api/models/{modelId}/simulate")
 	@Consumes(MediaType.APPLICATION_JSON)
 	Response simulate(
 		@PathParam("modelId") String modelId,
 		SimulateParams params
 	);
 
 	@POST
-	@Path("/model-composition")
+	@Path("/api/models/model-composition")
 	@Consumes(MediaType.APPLICATION_JSON)
 	Response modelComposition(
 		ModelCompositionParams params
 	);
 
 	@GET
-	@Path("/stratify/{modelA}/{modelB}/{typeModel}")
+	@Path("/api/models/stratify/{modelA}/{modelB}/{typeModel}")
 	@Consumes(MediaType.APPLICATION_JSON)
 	Response stratify(
 		@PathParam("modelA") String modelA,
 		@PathParam("modelB") String modelB,
 		@PathParam("typeModel") String typeModel
 	);
 
+	@POST
+	@Path("/api/petri-to-latex")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.TEXT_PLAIN)
```

Comment:
I cannot do `/api/models/petri-to-latex` on the Julia side, ti gets confused with `/api/models/:id`